### ISSUE TITLE
Videos that hang make fernschreiber hang

### ISSUE THREAD

**Issue by ApostolosB** [Wunderfitz/harbour-fernschreiber#492]
When you have a video in the chat you have two options: Play it in there or play it on a new page (tap the square icon). If you go to a new page and the video  doesn't load going back (swipe LtoR) makes Fern hang and give you the wait or close message.

I think i saw this issue (similar) with another app so it might be a general issue with how the videos are handled. But lets have it here also.

**Comment by Wunderfitz** [comment]
Yes, I have the same issue in Piepmatz. Couldn't find a fix so far - it's also hard to reproduce. Sometimes happens, in many cases it simply works properly. When it occurred while I had the debugger on I remember that something in the video drivers/Android adaption. If somebody has a hint, please let me know!

**Comment by ApostolosB** [comment]
Then i believe its something that must be reported to Jolla themselves.

**Comment by ApostolosB** [comment]
https://forum.sailfishos.org/t/video-player-used-in-apps-sometimes-makes-them-hang/11434

I filed it in the forums in case you want to chime in.
